src/system/data-transfer-tax-rates.md

```diff
@@ -46,7 +49,7 @@ If you conduct business in several states and ship a large quantity of product,
 
 ## Step 2: Prepare the Import Data
 
-You now have two spreadsheets open, side by side. One with the Magento export file structure, and the other with the new tax rate data that you want to import.
+You have two spreadsheets open, side by side. One with the Magento export file structure and the other with the new tax rate data that you want to import.
 
 1. To create a place to work in the spreadsheet with the new data, insert as many blank columns at the far left as needed and use cut-and-paste to rearrange the columns so they match the order of the Magento export data.
```

Comment:
I would want add columns to the far right, because the data is conditional/not required in all cases. YMMV

Reply:
i think it is an either or, but i'd probably do the same.  made the edit